src/ansible_navigator/_yaml.py

```diff
@@ -31,25 +31,24 @@ class YamlStyle(NamedTuple):
     allow_unicode: bool = True
 
 
-def human_dump(obj: Any, filename: str = None, file_mode: str = "w") -> Optional[str]:
+def human_dump(obj: Any, filename: Any = None, file_mode: str = "w") -> Optional[str]:
```

Comment:
Let's not introduce more `Any`s
```suggestion
def human_dump(
    obj: Any,
    filename: Union[io.TextIOWrapper, str] = None,
    file_mode: str = "w",
) -> Optional[str]:
```